dobject.py

```diff
@@ -624,168 +629,170 @@ class AddOnlySortedSet:
             self._send((y,))
-    
+
     def _send(self, els):
         if len(els) > 0:
             self._handler.send(dbus.Array([self._trans(el, True) for el in els]))
-    
+
     def _net_update(self, y):
         d = ListSet()
         d._list = y
-        d -= self._set
+        d -= self._list
         if len(d) > 0:
-            self._set |= d
+            self._list |= d
             self._trigger(d)
-    
+
     def receive_message(self, msg):
         self._net_update([self._trans(el, False) for el in msg])
-    
+
     def get_history(self):
-        if len(self._set._list) > 0:
-            return dbus.Array([self._trans(el, True) for el in self._set._list])
+        if len(self._list._list) > 0:
+            return dbus.Array([self._trans(el, True) for el in self._list._list])
         else:
             return dbus.Array([], type=dbus.Boolean) #prevent introspection of empty list, which fails
-    
+
     add_history = receive_message
-    
+
     def register_listener(self, L):
         """Register a listener L(diffset).  Every time another user adds items
         to the set, L will be called with the set of new items as a SortedSet."""
         self._listeners.append(L)
-        L(self._set.copy())
-    
+        L(self._list.copy())
+
     def _trigger(self, s):
         for L in self._listeners:
             L(s)
-    
+
     def __repr__(self):
-        return 'AddOnlySortedSet(' + repr(self._handler) + ', ' + repr(self._set) + ', ' + repr(self._trans) + ')'
-        
-        
+        return 'AddOnlySortedSet(' + repr(self._handler) + ', ' + repr(self._list) + ', ' + repr(self._trans) + ')'
+
+
 def CausalHandler():
     """The CausalHandler is analogous to the UnorderedHandler, in that it
     presents an interface with which to build a wide variety of objects with
     distributed state.  The CausalHandler is different from the Unordered in two
     ways:
-    
+
     1. The send() method of an CausalHandler returns an index, which must be
     stored by the CausalObject in connection with the information that was sent.
     This index is a universal, fully-ordered, strictly causal identifier
     for each message.
-    
+
     2. A CausalObject's receive_message method takes two arguments: the message
     and its index.
-    
+
     As a convenience, there is also
-    
+
     3. A get_index() method, which provides a new index on each call, always
     higher than all previous indexes.
-    
+
     CausalObjects are responsible for including index information in the
     return value of get_history, and processing index information in add_history.
-    
+
     It is noteworthy that CausalHandler is in fact implemented on _top_ of
     UnorderedHandler.  The imposition of ordering does not require lower-level
     access to the network.  This fact of implementation may change in the
     future, but CausalObjects will not be able to tell the difference.
     """
-    _max64 = 2**64
+    max64 = 2**64
```

Comment:
Is changed here but not where it is referenced.

Reply:
While fixing flake8, I removed an underscore on mistake, fixed on https://github.com/sugarlabs/stopwatch/pull/7/commits/0b3a93d263c37e29be58b9390d63a0c924c4c840